hoot-core/src/main/cpp/hoot/core/index/OsmMapIndex.cpp

```diff
@@ -495,7 +499,8 @@ boost::shared_ptr<const HilbertRTree> OsmMapIndex::getWayTree() const
   }
 
   OsmMapIndex* t = const_cast<OsmMapIndex*>(this);
-  for (set<long>::const_iterator it = _pendingWayInsert.begin(); it != _pendingWayInsert.end(); ++it)
+  for (set<long>::const_iterator it = _pendingWayInsert.begin(); it != _pendingWayInsert.end();
+       ++it)
```

Comment:
Why do you put line breaks like this?

Reply:
Ha.  Lines breaking at 100 cols was actually part of the original hoot formatting requirements to support those using vertical monitors, like me...a carryover from Signature Analyst development....but never really enforced.  It could probably be a little wider but 100 leaves enough room for the project view in the IDE etc and avoid having to scroll lines to the side.  I guess the only guys ever sticking to 100 cols were me and a guy that left awhile back.